lib/membrane_webm/parser/vint.ex

```diff
@@ -0,0 +1,379 @@
+defmodule Membrane.WebM.Parser.Vint do
+  @moduledoc """
+  VINT - variable length integer
+
+  Numbers are encoded as VINTs in EBML.
+  A VINT consists of three parts:
+  - VINT_WIDTH - the number N of leading `0` bits in the first byte of the VINT signifies how many bytes the VINT takes up in total: N+1; having no leading `0` bits is also allowed in which case the VINT takes 1 byte
+  - VINT_MARKER - the `1` bit immediately following the VINT_WIDTH `0` bits
+  - VINT_DATA - the 7*N bits following the VINT_MARKER
+  """
+
+  use Bitwise
+
+  # not sure if thte first `8` shouldn't be -inf or something. are all 0's legal?
+  @vint_width [
+    8,
```

Comment:
This can't stay that way, please put those in multi lines but not a single number per line.